openedx/adg/lms/registration_extension/models.py

```diff
@@ -14,7 +14,7 @@ class ExtendedUserProfile(models.Model):
 
     user = models.OneToOneField(
         User, unique=True, db_index=True, related_name='extended_profile', on_delete=models.CASCADE)
-    company = models.CharField(blank=True, null=True, max_length=50, choices=BUSINESS_LINES)
+    company = models.OneToOneField(BusinessLine, on_delete=models.CASCADE, null=True)
```

Comment:
Please also set blank=True

Reply:
It is a required field as well.

Reply:
If it is a required field can we remove `null=True`